cypress/support/fragments/settings/remote-storage/configurations.js

```diff
@@ -48,35 +59,66 @@ function saveAndCloseForm() {
   cy.wait(2000);
 }
 
+function openCreateConfigurationForm() {
+  cy.do(newButton.click());
+}
+
 const configurations = {
   DematicEMS: {
     title: 'Dematic EMS',
     create(name) {
+      openCreateConfigurationForm();
       fillGeneralInfo(name, this.title);
       saveAndCloseForm();
     },
   },
   DematicStagingDirector: {
     title: 'Dematic StagingDirector',
     create(name) {
-      fillGeneralInfo(name, this.title);
-      cy.do(configurationFields.timingInput.fillIn('1'));
+      openCreateConfigurationForm();
+      this.fillRequiredFields(name);
       saveAndCloseForm();
     },
+    fillRequiredFields(name, timing = '1') {
+      fillGeneralInfo(name, this.title);
+      cy.do(configurationFields.timingInput.fillIn(timing));
+    },
+    verifyRequiredFields(name, timing = '1') {
+      cy.expect(configurationFields.nameInput.has({ value: name }));
+      cy.expect(configurationFields.timingInput.has({ value: timing }));
+    },
   },
   CaiaSoft: {
     title: 'CaiaSoft',
     create(name) {
+      openCreateConfigurationForm();
+      this.fillRequiredFields(name);
+      saveAndCloseForm();
+    },
+    fillRequiredFields(
+      name,
+      accessionHoldingWorkflow = 'Change permanent location',
+      returningWorkflow = 'Items received at remote storage scanned into FOLIO',
+    ) {
       fillGeneralInfo(name, this.title);
       cy.do([
-        Accordion({ label: 'Accession holding workflow preference' })
-          .find(Select())
-          .choose('Change permanent location'),
-        Accordion({ label: 'Returning workflow preference' })
-          .find(Select())
-          .choose('Items received at remote storage scanned into FOLIO'),
+        configurationFields.accessionHoldingWorkflowDropdown.choose(accessionHoldingWorkflow),
+        configurationFields.returningWorkflowDropdown.choose(returningWorkflow),
       ]);
-      saveAndCloseForm();
+    },
+    verifyRequiredFields(
+      name,
+      accessionHoldingWorkflow = 'Change permanent location',
+      returningWorkflow = 'Items received at remote storage scanned into FOLIO',
+    ) {
+      expect(configurationFields.nameInput.has({ value: name }));
+      expect(configurationFields.provider.has({ value: including(this.title) }));
+      expect(
+        configurationFields.accessionHoldingWorkflowDropdown.has({
+          value: accessionHoldingWorkflow,
+        }),
+      );
+      expect(configurationFields.returningWorkflowDropdown.has({ value: returningWorkflow }));
```

Comment:
if multiple assertions in a row, better to pass as an array

Reply:
fixed